### ISSUE TITLE
Multiple "DO NOT EDIT" comments in etc/hosts

### ISSUE THREAD

**Issue by tokbes** [guumaster/hostctl#65]
**Describe the bug**
Every time when run `add`/`replace` commands `hostctl` is adding 1 more header 
```
##################################################################
# Content under this line is handled by hostctl. DO NOT EDIT.
##################################################################
```

**To Reproduce**
just run `hostctl replace -f /path/of/some/project/repo/.etchosts` few times

**Expected behavior**
don't add header if it's already exists

**Comment by guumaster** [comment]
Thanks for reporting this, I'll look into it.

**Comment by guumaster** [comment]
Fixed on version v1.0.11
